README.md

```diff
@@ -218,7 +218,7 @@ include on the number, defaulting to `3`, which provides microsecond precision.
 
 The status code of the response.
 
-If the request/response cycle completes before a response was sent to the
+If the request/response cycle ends before a response was sent to the
```

Comment:
This isn't a typo. If you still believe so, please elaborate.

Reply:
I will undo it